assets/javascripts/discourse/services/dropdown-buttons.js

```diff
@@ -35,24 +35,28 @@ export default class DropdownButtonsService extends Service {
     // case 3 - url starts with *, e.g. *example, it should match any url ending with "example"
     // case 4 - url ends with *, e.g. example*, it should match any url starting with "example"
 
-    const startsWithStar = url.startsWith("*");
-    const endsWithStar = url.endsWith("*");
+    // Do all string comparisons in lowercase.
+    const _url = url.toLowerCase();
```

Comment:
`#shouldHighlightByURL` is only used once. We can pass `url.toLoweCase()` in line `24` and we do not need to create this variable at all, the URL will already be in lowercase.

Reply:
good catch. I removed the `_url` variable, convert it to lowercase in line `24` as you suggested.